### DISCUSSION TITLE
Update from 1.12.2/102 to 1.3.3/23 not possible

### DISCUSSION THREAD

**Discussion by Nordlicht-13** [Akylas/OSS-DocumentScanner#339] [General]
Update from 1.12.2/102 to 1.3.3/23 not possible under Obtainium.
The downloaded Paket-ID not similar to App-ID - com.akylas.cardwallet [OSS Dokumentenscanner]

**Comment by farfromrefug** [comment]
@Nordlicht-13 That repo publishes 2 apps:
* OSS Document Scanner
* OSS Card Wallet

use the obtainium badges from the README to get it to work
